style.css

```diff
@@ -338,7 +358,322 @@ nav a {
+  }
+
+  .top {
+    margin-top: 80px;
+  }
+
+  .skills {
+    width: 364px;
+    display: flex;
+    flex-direction: column;
+    padding: 0;
+  }
+
+  .list {
+    display: flex;
+    gap: 20px;
+  }
+
+  .item {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    width: 122px;
+    height: 100px;
+    background: #f7f7f9;
+    border-radius: 10px;
+    padding: 10px 8px;
+  }
+
+  .item p {
+    margin-top: 4px;
+  }
+
+  .contact-me {
+    margin-top: -60%;
+    text-align: center;
+  }
+
+  .shapes {
+    min-height: 120vh;
+    background: url(./assets/desktop-shape.svg);
+    background-position: top right;
+    background-repeat: no-repeat;
+    background-size: contain;
+    padding: 20px 90px;
+  }
+
+  .shapes form {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .input-col {
+    width: 447px;
+    height: 48px;
+    background: #fff;
+    border: 1px solid #cfd8dc;
+    border-radius: 10px;
+    padding: 0 10px;
+    margin-bottom: 12px;
+    outline: none;
+    font-size: 17px;
+    font-weight: 400;
+    color: #172b4d;
+  }
+
+  .comment {
+    height: 170px;
+    padding: 12px;
+    resize: none;
+  }
+}
+
+@media only screen and (min-width: 1000px) {
+  header {
+    min-height: 160vh;
+    border-bottom-left-radius: 70px;
+  }
+
+  .link img {
+    display: none;
+  }
+
+  .link ul {
+    display: block;
+  }
+
+  .link ul li {
+    display: inline-block;
+    padding: 0 10px;
+  }
+
+  .link ul li a {
+    color: #344563;
+    font-size: 15px;
+    font-weight: 500;
+    cursor: pointer;
+  }
+
   .background {
-    background: none;
+    min-height: 130vh;
+    background: url("./assets/Header\ bg.svg");
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: contain;
+    margin-top: -100px;
+  }
+
+  .intro {
+    padding: 140px 250px;
+  }
+
+  .text {
+    font-size: 30px;
+    letter-spacing: 0.37px;
+    margin-top: 0;
+  }
+
+  .intro-text {
+    font-size: 18px;
+    line-height: 28px;
+  }
+
+  .projects {
+    margin-top: 80px;
+    gap: 60px;
+  }
+
+  .project-row {
+    display: grid;
+    width: 816px;
+    padding: 30px 60px;
+    grid-template-columns: repeat(2, auto);
+    grid-gap: 60px;
+  }
+
+  .project-row:nth-child(2n) .left {
+    order: 1;
+    margin-right: 80px;
+    margin-left: 0;
+    width: 240px;
+    height: 245px;
+    margin-top: 28px;
+  }
+
+  .left img {
+    width: 340px;
+    height: 245px;
+    margin-top: 28px;
+  }
+
+  .right h1 {
+    font-size: 40px;
+  }
+
+  .right .content {
+    font-size: 18px;
+  }
+
+  .right p {
+    font-size: 16px;
+  }
+
+  .about-me {
+    min-height: 140vh;
+    background: #fff;
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-gap: 30px;
+    justify-content: space-between;
+    border-top-right-radius: 60px;
+    padding: 120px 50px;
+    margin-top: 10%;
+    margin-bottom: 30%;
+    margin-right: 0;
+  }
```

Comment:
- [ ] As suggested by the previous reviewer please make sure to reduce the height of the "about me" section to around `80vh` as it's very long currently. ✔
- Previous comment:
> * [ ]  Kindly consider changing this to `min-height: 80vh;` as 140 makes your design different from the original design.
>       your design
>       ![qw](https://user-images.githubusercontent.com/96683843/236251702-8081f7fe-6edd-4c13-ac48-8778ac217434.jpg)
>       figma design
>       ![v](https://user-images.githubusercontent.com/96683843/236251796-6a83dc94-2467-4e02-9a57-86acce218c4e.jpg)

Reply:
Please I did reduced and it was like according to the figma design